File: task-03/Javascript/Sub-Task 2.js
const fs = require('fs');
fs.writeFile('input.txt', "hellljhcbaucjzcgvugvgvcugdsvugvugv", (err) => {
    if (err) {
        console.error('Error writing to input.txt:', err);
        return;
    }
    fs.readFile('input.txt', 'utf8', (err, data) => {
        if (err) {
            console.error('Error reading input.txt:', err);
            return;
        }
        fs.writeFile('output.txt', data, (err) => {
            if (err) {
                console.error('Error writing to output.txt:', err);
            } else {
                console.log('Data successfully written to output.txt');
            }
        });
    });
});

File: task-03/haskell/Sub-Task 2.hs
import System.IO

main :: IO ()
main = do
    let content="jhbuhebfurgfyugrwfryfgyurgfyrgf"
    writeFile "input.txt" content
    inputFile <- readFile "input.txt"
    writeFile "output.txt" inputFile
    outputFile <- readFile "input.txt"
    putStrLn "Content of output.txt:"          
    putStrLn outputFile  
